src/Widgets/AgendaEventRow.vala

```diff
@@ -299,6 +300,8 @@ public class Maya.View.AgendaEventRow : Gtk.ListBoxRow {
         string end_time_string = end_date.format (Settings.time_format ());
         string? datetime_string = null;
 
+        var is_same_time = start_date_string == end_date_string;
```

Comment:
Should be time string, not date string ;)

Reply:
Hahaha I'm so sorry!! Not my best day

Reply:
Happens to us all!